ruby-3.4.6-circleci/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM parkwhiz/rails-base:ruby-3.4.6_20250930095710
+LABEL org.opencontainers.image.authors="ParkWhiz <dev@parkwhiz.com>"
+
+WORKDIR /
+
+# ---------------------
+# -- Installing Docker.
+# ---------------------
+
+# https://docs.docker.com/engine/install/debian/
+RUN \
+  curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg && \
+  echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null && \
+  apt-get update && \
+  apt-get install -y --no-install-recommends postgresql-client-17 docker-ce docker-ce-cli cmake containerd.io wget && \
```

Comment:
I don't think this is gonna work with the current database version, you should use 16 until we're done with db upgrade.